mail/chrome/messenger/messenger.properties

```diff
@@ -315,6 +317,7 @@ editTagTitle = Edycja etykiety
 
 # For the virtual folder list dialog title %S is the name of the saved search folder
 editVirtualFolderPropertiesTitle = Edytuj właściwości folderu wyszukiwania %S
+virtualFolderSourcesChosen=#1 folder wybrany;Wybrano folderów: #1
```

Comment:
This is wrong, see https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals